wingtips-zipkin/src/main/java/com/nike/wingtips/zipkin/WingtipsToZipkinLifecycleListener.java

```diff
@@ -94,7 +99,14 @@ public void spanSampled(Span span) {
 
     @Override
     public void spanCompleted(Span span) {
-        zipkin.Span zipkinSpan = zipkinSpanConverter.convertWingtipsSpanToZipkinSpan(span, zipkinEndpoint, localComponentNamespace);
-        zipkinSpanSender.handleSpan(zipkinSpan);
+        try {
+            zipkin.Span zipkinSpan = zipkinSpanConverter.convertWingtipsSpanToZipkinSpan(span, zipkinEndpoint, localComponentNamespace);
+            zipkinSpanSender.handleSpan(zipkinSpan);
+        }
+        catch(Throwable ex) {
+            zipkinConversionOrReportingErrorLogger.warn("Found a span that is not zipkin compatible, or there was an error during span handling. "
```

Comment:
depending on env, this might lead to log statements killing a server (since warn is usually on). I'd recommend incrementing a counter, lowering the log level, or mentioning somewhere javadocs or otherwise that if you have untrusted callers disable logging on this component. (iotw, this is easy to abuse)